ironfish/src/fileStores/internal.ts

```diff
@@ -6,10 +6,12 @@ import { KeyStore } from './keyStore'
 
 export type InternalOptions = {
   isFirstRun: boolean
+  secretKey: string
```

Comment:
Our users might think this is their wallet secret key. What do you think about something more specific like `networkIdentity`?